react-demo-app/src/components/Video/ActiveVideoConversationsTable.tsx

```diff
@@ -0,0 +1,181 @@
+import { useDispatch, useSelector } from "react-redux";
+import { GuxButton, GuxRadialLoading, GuxTable } from "genesys-spark-components-react";
+import useSdk from "../../hooks/useSdk.ts";
+import {
+  IActiveVideoConversationsState, setCurrentlyDisplayedConversation,
+  toggleAudioMute,
+  toggleVideoMute, updateConversationMediaStreams
+} from "../../features/videoConversationsSlice.ts";
+import Card from "../Card.tsx";
+import './Video.css'
+import './ActiveVideoConversationsTable.css'
+import { VideoMediaSession } from "genesys-cloud-webrtc-sdk";
+import { AppDispatch, RootState } from "../../store.ts";
+
+export default function ActiveVideoConversationsTable() {
+  const videoConversations: IActiveVideoConversationsState[] = useSelector(
+    (state: RootState) => state.videoConversations.activeVideoConversations
+  );
+  const dispatch = useDispatch<AppDispatch>();
+  const { endSession, getSession } = useSdk();
+  const currentlyDisplayedConversationId = useSelector(
+    (state: RootState) => state.videoConversations.currentlyDisplayedConversationId
+  );
+
+  function getParticipantUsingDemoApp(index: number) {
+    const session = getSession(videoConversations[index].conversationId);
+    return videoConversations[index].participantsUpdate?.activeParticipants.find(
+      participant => session.fromUserId === participant.userId
+    );
+  }
+
+  function handleVideoMuteToggle(index: number) {
+    const participant = getParticipantUsingDemoApp(index);
+    if (participant && !participant.sharingScreen) {
+      dispatch(toggleVideoMute({
+        mute: !participant.videoMuted,
+        conversationId: videoConversations[index].conversationId,
+        userId: participant.userId
+      }));
+    }
+  }
+
+  function handleAudioMuteToggle(index: number) {
+    const participant = getParticipantUsingDemoApp(index);
+    if (participant) {
+      dispatch(toggleAudioMute({
+        mute: !participant.audioMuted,
+        conversationId: videoConversations[index].conversationId,
+        userId: participant.userId
+      }));
+    }
+  }
+
+  const handleConversationSwitch = (conversationId: string) => {
+    dispatch(setCurrentlyDisplayedConversation({ conversationId }));
+  };
+
+  const isLocalPartSharingScreen: boolean[] =
```

Comment:
NIT: just use the longform so its more clear - `isLocalParticipantScreensharing`